angular-ngc/libraries/lib-a/src/test.ts

```diff
@@ -0,0 +1,11 @@
+import { getTestBed } from '@angular/core/testing';
```

Comment:
I don't want to duplicate this file in every single library, there's nothing "lib-a" specific in here. Lets move this into `tools/testing` or something like that. See initialize_testbed.ts in the rules_nodejs example.